src/components/common/Bar.vue

```diff
@@ -4,9 +4,13 @@
       <slot />
     </p>
     <div class="right">
-      <button v-if="link" class="button" @click="goToLink(link)">
-        {{ linkCaption }}
-      </button>
+      <TmBtn
+        v-if="link && !session.developmentMode"
```

Comment:
```suggestion
        v-if="link"
```

Reply:
This is what @jbibla said.

"we should default to having no button and the staging bar doesn't need one"

Or maybe I misunderstood

Reply:
yeah, we can remove the link in the db again